src/Navbar.jsx

```diff
@@ -47,23 +47,36 @@ const Navbar = ({ token, deleteToken }) => {
         <StyledListItem>
           <StyledLink to="/add">Add Conference</StyledLink>
         </StyledListItem>
-        <StyledListItem>
-          <StyledLink to="/users/register">Register</StyledLink>
-        </StyledListItem>
-        <StyledListItem>
-          {token ? (
-            <Button
-              navLink
-              onClick={() => {
-                logout();
-              }}
-            >
-              Logout
-            </Button>
-          ) : (
-            <StyledLink to="/users/login">Login</StyledLink>
-          )}
-        </StyledListItem>
+        {sessionToken ? (
+          <>
```

Comment:
These aren't needed as there is only one component `StyledListItem` in this block, everything is a child of it

Reply:
likewise below

Reply:
Are you talking about the `<></>`? There are two `StyledListItem` components within each set of them for whether the `sessionToken` cookie is or isn't present so we thought we did need them?

Reply:
Yeah you're right, sorry i misread this. Didnt notice there were two `StyledListItems`